### ISSUE TITLE
Initate a call with bypassing the dialer interface

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#297]
```
Bug sur Nexus S/Android 4 (Compose le numéro mais ne passe pas l'appel)
```

Original issue reported on code.google.com by `jammesfl...@gmail.com` on 24 Sep 2012 at 7:06

Attachments:
- [PhoneManager.java.patch](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-297/comment-0/PhoneManager.java.patch)

**Comment by GoogleCodeExporter** [comment]
```
Hello

The purpose of this command is to dial without call the person (in order to 
have a manual check).

So, we can do:
- keep the dial command as it
- create a new command "call" that really do the call
```

Original comment by `Florent....@gmail.com` on 24 Sep 2012 at 8:49

**Comment by GoogleCodeExporter** [comment]
```
Hello,

Yes, this feature would be a real plus.

Congratulations for your work and responsiveness.

cordially


```

Original comment by `jammesfl...@gmail.com` on 24 Sep 2012 at 10:40

**Comment by GoogleCodeExporter** [comment]
```
Hello florent, 

CITE: create a new command "call" that really do the call

what is the command argument to call created via the interface?

Thank
```

Original comment by `romain.b...@gmail.com` on 24 Sep 2012 at 10:27

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Or cmd:service call phone 2 s16 "0123456789"

more info: 
http://davanum.wordpress.com/2007/12/12/android-calldial-from-the-adb-shell-comm
and-line/
```

Original comment by `romain.b...@gmail.com` on 24 Sep 2012 at 11:59

**Comment by GoogleCodeExporter** [comment]
```
We already have CALL_PHONE permission so something like

    Intent callIntent = new Intent(Intent.ACTION_CALL);
    callIntent.setData(Uri.parse("tel:0377778888"));
    startActivity(callIntent);

should initiate a call with bypassing the dialer interface.

Sources:
http://developer.android.com/reference/android/Manifest.permission.html#CALL_PHO
NE
http://www.mkyong.com/android/how-to-make-a-phone-call-in-android/
```

Original comment by `fschm...@gmail.com` on 25 Sep 2012 at 7:34
- Changed title: **Initate a call with bypassing the dialer interface**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
New command "call" commited
```

Original comment by `Florent....@gmail.com` on 1 Oct 2012 at 7:30
- Changed state: **Fixed**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 4.3 release
```

Original comment by `fschm...@gmail.com` on 26 Jan 2013 at 5:45
- Changed state: **Deployed**
